packages/react/src/AvatarStack/AvatarStack.module.css

```diff
@@ -134,23 +125,6 @@
     padding: 0.1px;
   }
 
-  &:nth-child(n + 3) {
-    --overlap-size: var(--overlap-size-avatar-three-plus);
-
-    /* stylelint-disable-next-line alpha-value-notation */
-    opacity: calc(100% - 2 * var(--opacity-step));
-  }
-
-  &:nth-child(n + 4) {
-    /* stylelint-disable-next-line alpha-value-notation */
-    opacity: calc(100% - 3 * var(--opacity-step));
-  }
-
-  &:nth-child(n + 5) {
-    /* stylelint-disable-next-line alpha-value-notation */
-    opacity: calc(100% - 4 * var(--opacity-step));
-  }
-
   &:nth-child(n + 6) {
```

Comment:
Maybe this can be variable? Consumers of AvatarStack decide on the max amount of avatars to show at rest 🤔